_data/simple-icons.json

```
@@ -4316,6 +4316,11 @@
             "hex": "2ECE53",
             "source": "https://github.com/docsifyjs/docsify/blob/d01841fd9d829adeeae6cde1d5818ce8798c7e58/docs/_media/icon.svg"
         },
+        {
+            "title": "Doctrine",
+            "hex": "FA6A3C",
+            "source": "https://www.doctrine-project.org/logos/doctrine-logo.svg"
```

We don't need to link to the specific file if it's not on a repository - more likely to 404 if they rebrand!

```suggestion
            "source": "https://www.doctrine-project.org"
```